habitat/config/default.py

```diff
@@ -93,11 +93,11 @@ def __init__(self, *args, **kwargs):
 # -----------------------------------------------------------------------------
 # # COMPOSITE TASK CONFIG
 # -----------------------------------------------------------------------------
-_C.TASK.TASK_SPEC_BASE_PATH = "configs/tasks/rearrange/pddl/"
+_C.TASK.TASK_SPEC_BASE_PATH = "habitat/config/tasks/rearrange/pddl/"
```

Comment:
To avoid this issues in future:
```suggestion
_C.TASK.TASK_SPEC_BASE_PATH = f"{DEFAULT_CONFIG_DIR}/tasks/rearrange/pddl/"
```